{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -39,5 +39,154 @@ def build_artifacts(location: str, output_dir: str, append: bool, verbose: int):
+        called by the :func:`build_artifacts` function located in the same
+        module.
+
+
+    """
+    from vivarium_cluster_tools.psimulate.utilities import get_drmaa
+    drmaa = get_drmaa()
+
+    jobs = {}
+    with drmaa.Session() as session:
+        for location in project_globals.LOCATIONS:
+            path = output_dir / f'{sanitize_location(location)}.hdf'
+
+            job_template = session.createJobTemplate()
+            job_template.remoteCommand = shutil.which("python")
+            job_template.args = [__file__, str(path), f'"{location}"']
+            job_template.nativeSpecification = (f'-V '  # Export all environment variables
+                                                f'-b y '  # Command is a binary (python)
+                                                f'-P {project_globals.CLUSTER_PROJECT} '  
+                                                f'-q {project_globals.CLUSTER_QUEUE} '  
+                                                f'-l fmem={project_globals.MAKE_ARTIFACT_MEM} '
+                                                f'-l fthread={project_globals.MAKE_ARTIFACT_CPU} '
+                                                f'-l h_rt={project_globals.MAKE_ARTIFACT_RUNTIME} '
+                                                f'-l archive=TRUE '  # Need J-drive access for data
+                                                f'-N {sanitize_location(location)}_artifact')  # Name of the job
+            jobs[location] = (session.runJob(job_template), drmaa.JobState.UNDETERMINED)
+            logger.info(f'Submitted job {jobs[location][0]} to build artifact for {location}.')
+            session.deleteJobTemplate(job_template)
+
+        decodestatus = {drmaa.JobState.UNDETERMINED: 'undetermined',
+                        drmaa.JobState.QUEUED_ACTIVE: 'queued_active',
+                        drmaa.JobState.SYSTEM_ON_HOLD: 'system_hold',
+                        drmaa.JobState.USER_ON_HOLD: 'user_hold',
+                        drmaa.JobState.USER_SYSTEM_ON_HOLD: 'user_system_hold',
+                        drmaa.JobState.RUNNING: 'running',
+                        drmaa.JobState.SYSTEM_SUSPENDED: 'system_suspended',
+                        drmaa.JobState.USER_SUSPENDED: 'user_suspended',
+                        drmaa.JobState.DONE: 'finished',
+                        drmaa.JobState.FAILED: 'failed'}
+
+        if verbose:
+            logger.info('Entering monitoring loop.')
+            logger.info('-------------------------')
+            logger.info('')
+
+            while any([job[1] not in [drmaa.JobState.DONE, drmaa.JobState.FAILED] for job in jobs.values()]):
+                for location, (job_id, status) in jobs.items():
+                    jobs[location] = (job_id, session.jobStatus(job_id))
+                    logger.info(f'{location:<35}: {decodestatus[jobs[location][1]]:>15}')
+                logger.info('')
+                time.sleep(10)
```

Comment:
drop in the sleep constant here!

Reply:
Had it on the clipboard and got distracted...

Reply:
Done.